GettingStarted-Advanced.md

```diff
@@ -1,3 +1,21 @@
+The "Advanced" setup is the set of final steps to capture your application preferences (both system apps as well as custom apps) and back them up into an *encrypted remote repository*. Currently this kind of a service is offered by [keybase](https://keybase.io/) where you can get private, fully-encrypted repos for free.
```

Comment:
Not sure I agree that these lines need to move into the advanced section. "Advanced" is basically starting with the home folder as a git repo and the profiles folder as another git repo. Anything before that, I would consider, as part of the basic / common setup. wdyt?

Reply:
intend was to keep the basic setup setup instructions as minimal as possible, but yeah i agree

Reply:
I agree that this can be the starting para in the Advanced file.